server/session/TransactionImpl.java

```diff
@@ -1178,6 +1180,10 @@ public void commit() throws TransactionException {
         }
     }
 
+    public void disableCommitValidation() {
```

Comment:
did we have a discussion on how we can do validation since its all disabled? I think this needs some more thought

Reply:
Uh, did I actually use this anywhere? I think I added it and then ended up not using it.

Reply:
you do call it in `newTransaction()`

Reply:
No, I'm wrong, we do use it. I'm pretty sure all this is only disabling the role player count validation for us, not removing all validation.